mqr/midline_ussd.py

```diff
@@ -0,0 +1,689 @@
+import logging
+from urllib.parse import urljoin
+
+import aiohttp
+
+from mqr import config
+from vaccine.base_application import BaseApplication
+from vaccine.states import Choice, ChoiceState, EndState
+from vaccine.utils import HTTP_EXCEPTIONS, normalise_phonenumber
+
+logger = logging.getLogger(__name__)
+# TODO: FWB
+
+
+def get_eventstore():
+    # TODO: Cache the session globally. Things that don't work:
+    # - Declaring the session at the top of the file
+    #   You get a `Timeout context manager should be used inside a task` error
+    # - Declaring it here but caching it in a global variable for reuse
+    #   You get a `Event loop is closed` error
+    return aiohttp.ClientSession(
+        timeout=aiohttp.ClientTimeout(total=5),
+        headers={
+            "Authorization": f"Token {config.EVENTSTORE_API_TOKEN}",
+            "Content-Type": "application/json",
+            "User-Agent": "mqr-baseline-study-ussd",
+        },
+    )
+
+
+def get_rapidpro():
+    return aiohttp.ClientSession(
+        timeout=aiohttp.ClientTimeout(total=5),
+        headers={
+            "Authorization": f"Token {config.RAPIDPRO_TOKEN}",
+            "Content-Type": "application/json",
+            "User-Agent": "mqr-midline-study-ussd",
+        },
+    )
+
+
+class Application(BaseApplication):
+    START_SURVEY = "state_start"
```

Comment:
This should stay START_STATE and you can change it to state_eat fruits

Reply:
Not 100% sure if I understood you correctly, please double check if I got it right